src/vm.rs

```diff
@@ -415,6 +413,98 @@ impl TestContextObject {
     }
 }
 
+/// Trace analyzer
+#[derive(Debug)]
```

Comment:
I don't think all these `Cow` wrappings are necessary here

Reply:
This allows us to have owned `TraceAnalyzer` as well as borrowing. I personally need this because I have to send `TraceAnalyzer` to another thread in plugin implementation but I don't want to affect performance of single-threaded code, so in such cases we just borrow these structures instead of cloning.